src/pilot/register_overcloud.py

```diff
@@ -344,8 +345,6 @@ def _get_consumed_pool_ids(self, node_ip):
         return consumed_pool_ids
 
     def _attach_node(self, role, node_ip):
-        if self._using_activation_key(role):
```

Comment:
This change removes support for using activation keys with CDN. Do we really want to do that?